server/middleware/auth.coffee

```diff
@@ -179,7 +179,13 @@ module.exports =
     
     activity = req.user.trackActivity 'login', 1
     yield req.user.update {activity: activity}
-    res.redirect '/'
```

Comment:
Should the clever login logic also use this logic of sending the user somewhere more specific then the homepage?  Should this logic be the auth middleware redirectHome / redirectAfterLogin?